Controlling input group validation programatically

guizoxxv opened the pull request:
Added a section about controlling input group validation programatically without the `was-validated` class. Discussed in this issue - https://github.com/twbs/bootstrap/issues/32340.

I did not openned the PR in the main branch because this file does not exists in https://github.com/twbs/bootstrap/tree/main/site/content/docs/5.0/components.

Preview: https://deploy-preview-32361--twbs-bootstrap.netlify.app/docs/4.5/components/forms/#controlling-input-group-validation-programatically

guizoxxv committed:
Controlling input group validation programatically

Added a section about controlling input group validation programatically without the `was-validated` class. Discussed in this issue - https://github.com/twbs/bootstrap/issues/32340.

XhmikosR commented:
Please make sure tests pass before pushing.

XhmikosR commented:
And also, if this affects v5, the PR should be against the main branch, not v4-dev. We will backport it ourselves if needed.

guizoxxv committed:
Fix duplicate IDs on input group validation

Fix duplicate IDs on section - Controlling input group validation programatically
